core/src/main/java/com/emc/ia/sdk/sip/client/ArchiveClient.java

```diff
@@ -94,4 +97,12 @@
    */
   OrderItem exportAndWait(SearchResults searchResults, ExportConfiguration exportConfiguration, String outputName, long timeOutInMillis) throws IOException;
 
+  /**
+   * Upload the transformation zip file with the stylesheet.
+   * @param exportTransformation The export transformation.
+   * @param zipFile The zip file with stylesheet.
+   * @throws IOException When an I/O error occurs
+   */
```

Comment:
Please add missing `@return`. You can see this on the command-line when building:

    /home/remon/workspace/git/infoarchive-sip-sdk/core/src/main/java/com/emc/ia/sdk/sip/client/ArchiveClient.java:106: warning: no @return
      LinkContainer uploadTransformationFile(ExportTransformation exportTransformation, File zipFile) throws IOException;
                    ^
    1 warning